packages/input-option/src/InputOption/InputOption.tsx

```diff
@@ -9,51 +9,100 @@ import {
 } from '@leafygreen-ui/polymorphic';
 
 import {
-  inputOptionActiveStyles,
-  inputOptionDisabledStyles,
-  inputOptionHoverStyles,
-  inputOptionStyles,
-  inputOptionThemeStyles,
+  baseStyles,
+  boldTitleStyles,
+  disabledStyles,
+  formWedgeStyles,
+  getContextStyles,
+  getHoverStyles,
   inputOptionWedge,
-  titleSelectionStyles,
+  menuTitleStyles,
+  menuWedgeStyles,
+  wedgeActiveStyles,
 } from './InputOption.style';
-import { InputOptionProps } from './InputOption.types';
+import {
+  ActionType,
+  InputOptionProps,
+  RenderedContext,
+  State,
+} from './InputOption.types';
 
-export const InputOption = Polymorphic<InputOptionProps>(
+export const InputOption = Polymorphic<InputOptionProps, 'div'>(
   (
     {
-      as = 'li' as PolymorphicAs,
+      as = 'div' as PolymorphicAs,
```

Comment:
Do you wanna group these props by defaults and aliases? I know you're a fan of that!